webpack/components/QuestionWidget.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+var routes = require("../routes.js");
+if(routes.Routes){ routes=routes.Routes;}
```

Comment:
The reason that is in there is because what gets returned by the require statement is different  between the mocha testing and everything else.  Mocha returns an object with the Routes property , everything else returns the Routes object itself.   

If this gets abstracted out to be passed in then it would need to be abstracted all the way out to what ever is including it, or anything else for that matter.  Maybe another approach is to have an intermediate lib that does the require and can make the determination in a central location.